examples/src/status-message-borsh.js

```diff
@@ -0,0 +1,40 @@
+import {NearBindgen, call, view, near, borshSerialize, borshDeserialize} from "near-sdk-js";
+
+const schema = {
+    struct: { records: {map: { key: 'string', value: 'string' }} }
+};
+
+function borshSerializeStatusMessage(statusMessage) {
+    return borshSerialize(schema, statusMessage);
+}
+
+function borshDeserializeStatusMessage(value) {
+    return  borshDeserialize(schema, value)
+}
+
+@NearBindgen({
+    serializer(value) {
+        return borshSerializeStatusMessage(value);
+    },
+    deserializer(value) {
+        return borshDeserializeStatusMessage(value);
```

Comment:
Nice! Not related of this PR, but to ensure `@NearBindgen(... serializer)` API does work, can you try if intentionally throw error here, the test will fail due to the error?

Reply:
add throw Error test

Reply:
Perfect!